build-tests/heft-webpack5-everything-test/src/test/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I think ideally we should test this in isolation, not in a test project that's also exercising something else.